shim/src/glue.c

```diff
@@ -1,23 +1,38 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
-
+#define _GNU_SOURCE
 #include <demi/libos.h>
 #include <demi/sga.h>
 #include <demi/wait.h>
 
-static __thread int __demi_reent_guard = 0;
+#include <string.h>
```

Comment:
Why do we need `string.h`?

Reply:
That was accidentally left behind when I was making some temporary modifications. It has been removed now.